### PR TITLE
[Doc] Add *Netty source connector guide*

### PR DESCRIPTION
**Pull request by Anonymitaet**
Fix https://github.com/apache/pulsar/issues/5015

### CODE REVIEW

File: site2/docs/io-netty-source.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    $ ./bin/pulsar-client consume -t Exclusive -s netty-sub netty-topic -n 0
+    ```
+
+7. Open another terminal window to send data to the Netty source.
+
+    ```bash
+    $ docker exec -it pulsar-netty-standalone /bin/bash
+    
+    $ curl -X POST --data 'hello, world!' http://127.0.0.1:10999/
+    ```
```
Comment (Anonymitaet) [inline site2/docs/io-netty-source.md]: @tuteng Is step 7 correct?

Comment (tuteng) [reply site2/docs/io-netty-source.md]: Yes, it is correct.

**Comment by tuteng** [comment]
run integration tests

**Comment by Anonymitaet** [comment]
@merlimat Thank you

**Pull request merged**